appstudio-controller/controllers/appstudio.redhat.com/environment_controller_test.go

```diff
@@ -1426,3 +1623,16 @@ var _ = Describe("Environment controller tests", func() {
 
 	})
 })
+
+func ExpectEnvironmentConditionErrorOccured(envMessage string, env appstudioshared.Environment) {
```

Comment:
A utility function to reduce code duplication of some of the other tests in this package.